pkg/client/metrics/metrics.go

```diff
@@ -14,54 +14,46 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// Package metrics provides utilities for registering client metrics to Prometheus.
+// Package metrics provides abstractions for registering metrics to be recorded.
```

Comment:
How about:
`Package metrics provides abstractions for registering which metrics to record.`
I think that says the same thing.